test/rules/prefer_mixin_test.dart

```diff
@@ -8,17 +8,76 @@ import '../rule_test_support.dart';
 
 main() {
   defineReflectiveSuite(() {
-    defineReflectiveTests(PreferMixinTest);
+    defineReflectiveTests(PreferMixin30Test);
+    defineReflectiveTests(PreferMixin219Test);
   });
 }
 
-@reflectiveTest
-class PreferMixinTest extends LintRuleTest {
+abstract class BaseTest extends LintRuleTest {
```

Comment:
In the analyzer this would be `PreferMixinTestCases`, but it is a mixin. I don't remember why we decided to make these mixins. A base class like this would work too.

But naming it just `BaseTest` seems not ideal, you would get many of these. OTOH, this is a short nice name, easy to find once you open a file.

Reply:
renamed.  Thanks!